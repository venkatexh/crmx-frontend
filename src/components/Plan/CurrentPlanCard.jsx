import '../../sass/components/plan/planCard.component.scss';
import {useSelector} from "react-redux";

const CurrentPlanCard = () => {
  const state = useSelector(({loggedUser}) => ({loggedUser}));
  const {plan} = state.loggedUser;
  return (
    <div className={'currentPlan'}>
      <div className={'currentLeft'}>
        <div className={'textTop'}>Current Plan</div>
        <div className={'textMid'}>{plan.name}</div>
        <div className={'textBottom'}>${plan.amount} {plan.rate}</div>
      </div>
      <div className={'currentRight'}></div>
    </div>
  )
}

export default CurrentPlanCard;