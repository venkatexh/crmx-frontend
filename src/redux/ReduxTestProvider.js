import { Provider } from "react-redux";

const ReduxTestProvider = ({ children, reduxStore }) => (
  <Provider store={reduxStore}>{children}</Provider>
);

export default ReduxTestProvider;
