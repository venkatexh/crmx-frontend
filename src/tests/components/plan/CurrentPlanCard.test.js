import { shallow } from "enzyme";
import React from "react";
import CurrentPlanCard from "../../../components/Plan/CurrentPlanCard";
import ReduxTestProvider from "../../../redux/ReduxTestProvider";

it("Current plan card component snapshot", () => {
  expect(
    shallow(
      <ReduxTestProvider>
        <CurrentPlanCard />
      </ReduxTestProvider>
    )
  ).toMatchSnapshot();
});
