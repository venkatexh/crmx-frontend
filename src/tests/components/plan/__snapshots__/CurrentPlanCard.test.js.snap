// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Current plan card component snapshot 1`] = `ShallowWrapper {}`;
